Sources/SotoCore/AWSClient.swift

```diff
@@ -62,6 +61,8 @@ public final class AWSClient {
 
     /// default logger that logs nothing
     public static let loggingDisabled = Logger(label: "AWS-do-not-log", factory: { _ in SwiftLogNoOpLogHandler() })
+    /// default baggage context
+    public static let emptyContext = DefaultContext.TODO(logger: loggingDisabled)
```

Comment:
This used as the default context when no context is provided

Reply:
I am keen to keep a default context as this library isn't only used on server applications. It is also used in macOS and iOS client applications and I don't think the tracing context is as useful in those situations. Also I would suggest Soto is less glue than something like the http client.

I'm happy to change the `let` to the suggested `func`.
I'll have a think about the naming, not completely with `.todo` here given the possiblity of this being used as a client lib.

Reply:
I see, thanks for the reminder. I guess you might want to use `.topLevel` then after all. 
It's "more okey" for this lib than httpclient to provide a default here, feel free to stick with the default 👍

Reply:
Updated `emptyContext` to function, renamed to `defaultBaggageContext`. I'm going to keep it as a TODO because people will want to catch when they aren't passing context to soto.